packages/next-env/index.ts

```diff
@@ -157,6 +161,7 @@ export function loadEnvConfig(
       cachedLoadedEnvFiles.push({
         path: envFile,
         contents,
+        env: {}, // This will be populated in processEnv
```

Comment:
Not really following this comment, can you explain why adding this wil help showing "Loaded from" in TS doc in PR description